spec/useLoadClientScript.spec.tsx

```diff
@@ -33,8 +34,8 @@ describe("useLoadClientScript", () => {
 
     hook.rerender()
     expect(hook.result.current.clientScriptLoaded).toBe(true)
-    expect(window.nosto).toBeDefined()
-    expect(getScriptSources()).toEqual([`http://connect.nosto.com/include/${testAccount}`])
+    expect(getNostoWindow()).toBeDefined()
```

Comment:
should we expose isLoaded() instead ?

Reply:
It was already exposed, I guess I just forgot to use it here. Replaced. 🙂